packages/platform-ios/native_modules.rb

```diff
@@ -42,7 +42,11 @@ def use_native_modules!(root = "..", packages = nil)
       existing_dep.name.split('/').first == spec.name
     end
 
-    pod spec.name, :path => File.dirname(podspec_path)
+    # Use relative path
+    absolute_path = File.dirname(podspec_path)
+    relative_path = '../' + absolute_path.partition('node_modules').last(2).join()
```

Comment:
The `../` is a root here. It can be set to something different, e.g. `../..` in monorepos or libraries with examples. It's available from `root` variable. Let's change it to something like:
```rb
relative_podspec_path = File.join(root, File.dirname(podspec_path).partition('node_modules').last(2).join())
```

Reply:
thanks! completely forgot to take other structures into account, fixed